contrib/src/test/java/com/datatorrent/contrib/hbase/HBasePOJOInputOperatorTest.java

```diff
@@ -128,7 +144,7 @@ public void populateDAG(DAG dag, Configuration conf)
         Thread.sleep(1000);
       }
       catch( Exception e ){}
-      
+      logger.info("Tuple row key: ", output.getReceivedTuples());
       logger.info( "Received tuple number {}, instance is {}.", output.getReceivedTuples() == null ? 0 : output.getReceivedTuples().size(), System.identityHashCode( output ) );
       if( output.getReceivedTuples() != null && output.getReceivedTuples().size() == TUPLE_NUM )
```

Comment:
If this condition is never met, the unit test case will run forever. Need some timeout based failure of the test case.